data/json/mapgen/musicstore.json

```diff
@@ -38,7 +38,7 @@
           "s_music"
         ],
         "method": "json",
-        "weight": 180, "comment": "<-- Real decent place for a quick morale boost, relatively rare.",
+        "weight": 50, "comment": "<-- Real decent place for a quick morale boost, relatively rare.",
```

Comment:
Does this really work that way, I though this only controls the weight of this specific mapgen (and because there is only one mapgen for s_music, it doesn't matter at all)?

I think the data in data/json/regional_map_settings.json controls the frequency of those overmap terrains. s_music seems to have the same probability as a gas station.